assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -208,42 +208,22 @@ const MapElementsNoSelection = ({
 const RoutePatternLayers = ({
   routePattern,
   isSelected,
-  onShapeClick,
 }: {
   routePattern: RoutePattern
   isSelected: boolean
-  onShapeClick: () => void
 }): JSX.Element => {
   return routePattern.shape ? (
     <>
       <ZoomLevelWrapper>
         {(zoomLevel) => {
-          const routePatternIdSplit = routePattern.id.split("-")
-          const variantFormatted =
-            routePatternIdSplit.length === 3 && routePatternIdSplit[1] !== "_"
-              ? "_" + routePatternIdSplit[1]
-              : ""
           return (
             <>
               {routePattern.shape && (
                 <>
                   <RouteShape
                     shape={routePattern.shape}
                     isSelected={isSelected}
-                    onClick={onShapeClick}
-                  >
-                    {(zoomLevel >= 16 && !isSelected && (
-                      <Tooltip
-                        className="route-shape__tooltip"
-                        sticky={true}
-                        direction="top"
-                      >
-                        Click to select route {routePattern.routeId}
-                        {variantFormatted}.
-                      </Tooltip>
-                    )) ||
-                      undefined}
-                  </RouteShape>
```

Comment:
question: Is there styling that should be removed since this shape is no longer clickable? There is a pointer cursor when hovering over the shape

Reply:
I believe this comes from leaflet adding the `leaflet-interactive` class and I'm not sure how to get rid of it yet, but I can look into that.

Reply:
Thanks, that seems to work!

What do you think of c19ba3552532523bf5536e6c1c1eb55729efd0e9 ?
I was unsure if we should simply set it to `false` always or make it possible for the interactive state to be set based on if `onClick` is bound.

Reply:
Personally, I don't see a reason to remove the functionality from the component api (similar reason to why I didn't remove it originally before https://github.com/mbta/skate/pull/2096#discussion_r1232329359), but as a part of the larger systems of maps, I do see that this could essentially be tech debt if we don't use it. Removed in 3d9142de9a95f9ad700d42e8b2d336db5bc78226